sys/androidmedia/gstamc-constants.h

```diff
@@ -86,7 +86,12 @@ enum
   COLOR_FormatRawBayer8bitcompressed = 32,
   COLOR_FormatL2 = 33,
   COLOR_FormatL4 = 34,
-  COLOR_FormatL8 = 35,
+
+  /* Another collision: surface of omxrealtekvideodecoder.
+     COLOR_FormatL8 is never used in the code. */
+  /*  COLOR_FormatL8 = 35, */
+  COLOR_FormatSurface6 = 0x23,
```

Comment:
other members of this enum are also hex values sometimes. For example,
HAL_PIXEL_FORMAT_YCrCb_420_SP = 0x11, /* = 17 in dec. */

it's written like this because it's defined this way in Android's headers.

COLOR_FormatSurface6 is in hex because it was logged like this.

There're lots of other values in hex in this enum